index.html

```diff
@@ -42,8 +42,8 @@
         <div class="row">
             <div class="col-md-12">
                 <div class="col-md-8 col-md-offset-2">
-                <h4 class="jumbotron-text">Project Jupyter exists to develop open-source software, open-standards,
-                    and services for interactive computing across dozens of programming languages.</h4>
+                <h4 class="jumbotron-text">Project Jupyter develops open-source software, open standards
+                    and interactive services for dozens of computer programming languages.</h4>
```

Comment:
Thank you for the attempts to streamline the language. In this case I believe the meaning would change markedly (what are _interactive services_?); in my opinion `interactive computing` is a very specific term which should be kept as-is, and addition of  "computer" before "programming languages" is not necessary. Thanks!

```suggestion
                    and services for interactive computing across dozens of programming languages.</h4>
```